pkg/clients/pagerduty.go

```diff
@@ -140,45 +140,23 @@ func (c *PagerdutyClient) GetIncident(f *Filter) (*pagerduty.Incident, error) {
 }
 
 // AcknowledgeIncident sets a incident to status acknowledged and assigns the given user to it.
-func (c *PagerdutyClient) AcknowledgeIncident(incidentID string, user *pagerduty.User) error {
```

Comment:
Why is this block removed?
The code being removed ensures:
- timestamp of acknowledgement is saved
- multiple users can acknowledge and are assigned accordingly to the incident

Reply:
true (guess the format is relevant for Supernova) - need to check on that, my focus was on make it functional again